handler/src/test/java/io/netty/handler/ssl/ConscryptJdkSslEngineInteropTest.java

```diff
@@ -73,4 +73,10 @@ public void testMutualAuthValidClientCertChainTooLongFailOptionalClientAuth() th
     @Override
     public void testMutualAuthValidClientCertChainTooLongFailRequireClientAuth() throws Exception {
     }
+
+    @Override
+    protected boolean mySetupMutualAuthServerIsValidServerException(Throwable cause) {
```

Comment:
The fact that we didn't need it before was a bug. Conscrypt wasn't properly propagating the alert back to the client. Now it behaves the same way as netty-tcnative.